packages/providers/onekey-cosmos-provider/src/OnekeyCosmosProvider.ts

```
@@ -266,12 +266,14 @@
         id: string;
       };
       if(!checkWalletSwitchEnable()) return;
-
-      if (data && data.type === 'proxy-request' && data.method) {
+      const hasHandle = data && data.type && data.type.startsWith('proxy-request') && data.type !== 'proxy-request-response';
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)